dev/ansible/00_create_intention.yml

```diff
@@ -0,0 +1,34 @@
+- name: set variable
+  set_fact:
+    mode: "{{ mode }}"
```

Comment:
Is it necessary to create this variable `mode` ? I think it is unused?

Reply:
This is new! I agree with HM